### ISSUE TITLE
Participants of the tests

### ISSUE THREAD

**Issue by hannesrunersjo** [Simumatik/usability-test#2]
Here we can document and discuss the target audience for the tests.

**Comment by hannesrunersjo** [comment]
Here are some methods to recruit participants. (Remote focused)

-Recruiting participants from a panel:
This is perfect for testing usability of the mock-ups and prototypes, performing benchmark studies, testing information architecture, and many, many others. 

-Recruiting Participants from your Site users: 
My personal favourite – down the line it is the actual end users’ perception that counts. In my personal opinion this type of the remote test should not take longer than 5 minutes to complete. User should be able to answer to up to 15 questions in this time slot. These should be the key “burning” problems. The disadvantage of this method is that collecting the data can sometimes take weeks. Especially if the website generates little traffic or if we have a handful of customers. 

-Recruiting Participants from your Site users – Competition
You will have to offer a reward for completing the test/survey. Make sure that the prize is interesting enough to convince your users to participate. We can offer free upgrade just as the competition we had this summer,, a discount or another prize. In this case engage with Andy. He will help us to run the competition, prepare the catchy slogans and advise where necessary.

All though I prefer to have the test in a physical lab, due to corona It could be smart to have a remote test in mind.

**Comment by MikelAyani** [comment]
Probably a mix of what you comment above, but what if we offer making the survey directly from the app, so the actual users could see an entry on the "Release notes" and we offer a discount code for annual explorer subscription or a free Explorer month?

**Comment by hannesrunersjo** [comment]
Yeah absolutely, that is a good combo! That way we (probably) get relevant end users, and we offer something to increase the motivation to participate.
